packages/frontend/src/standalone-entrypoint.tsx

```diff
@@ -89,7 +93,10 @@ root.render(
     </StrictMode>,
 );
 
-if (__ENVIRONMENT__ === "prod" && "serviceWorker" in navigator) {
+if (
+    __ENVIRONMENT__ !== Environment.Production &&
```

Comment:
Here `__ENVIRONMENT__` should be equal to `Environment.Production`, no?

Reply:
Good point but I actually opted to only disable the service worker in the development environment, in line with the philosophy of having a staging environment that is as close as possible to production. Addressed in 8283f77c1cde43320b42b180ce930ddd13c721ea